dummy-api/matrix-service-api.yaml

```diff
@@ -0,0 +1,115 @@
+swagger: '2.0'
+info:
+  title: HCA Matrix Service
+  version: 0.0.0-dummy
+schemes:
+  - http
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  api_key:
+    type: apiKey
+    name: x-api-key
+    in: header
+definitions:
+  MatrixRequest:
+    type: object
+    properties:
+      bundle_fqids:
+        type: array
+        items:
+          type: string
+          format: dcp-fqid
+      bundle_fqids_url:
+        type: string
+        format: url
+      filter:
+        $ref: '#/definitions/MatrixFilter'
+      format:
+        $ref: '#/definitions/MatrixFormat'
+  MatrixFilter:
+    type: object
+    properties:
+      pandas_filter_string:
+        type: string
+  MatrixFormat:
+    type: string
+    enum:
+      - zarr
+      - parquet
+      - loom
+      - anndata
+      - singlecellexperiment
+  MatrixResponse:
+    type: object
+    properties:
+      request_id:
+        type: string
+        format: uuid
+      status:
+        $ref: '#/definitions/MatrixRequestStatus'
+      key:
+        type: string
+  MatrixRequestStatus:
+    type: string
+    enum:
+      - Initialized
+      - Running
+      - Complete
+      - Failed
```

Comment:
How does an application diagnose _Failed_ state?

Reply:
So I added a "message" to the response. Think that's enough?